.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,21 @@
-<!-- please add a icon to the title of this PR and delete this line and similar ones -->
-<!-- the icon will be either ⚠️ (:warning:, major or breaking changes), ✨ (:sparkles:, feature additions), 🐛 (:bug:, patch and bugfixes), 📖 (:book:, documentation or proposals), or 🌱 (:seedling:, minor or other) -->
+<!-- Thank you for contributing to Metal3! -->
+
+<!-- STEPS TO FOLLOW:
+	1.  Add an icon to the title of this PR (see https://sigs.k8s.io/cluster-api/CONTRIBUTING.md#contributing-a-patch), and delete this line and similar ones. The icon will be either ⚠️ (:warning:, major or breaking changes), ✨ (:sparkles:, feature additions), 🐛 (:bug:, patch and bugfixes), 📖 (:book:, documentation or proposals), or 🌱 (:seedling:, minor or other)
```

Comment:
why are we pointing to CAPI contributing guide?

Reply:
I copy pasted from CAMP3. So it was there before the changes. 
Do you want me to remove it?

Reply:
I think, we should point to our own contributing guide, but I am in the process of consolidating those. Leave it for now, I will do it as part of that PR.